src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2815,6 +2812,9 @@ private int[] SerializeMethodBodies(BlobBuilder ilBuilder, PdbWriter pdbWriterOp
 
             var encoder = new MethodBodiesEncoder(ilBuilder);
 
+            var mvidStringHandle = default(UserStringHandle);
```

Comment:
I prefer the current 2 arguments over KeyValuePair since we can use names. A tuple with named items would of course be ideal.

---
In reply to: [63577698](https://github.com/dotnet/roslyn/pull/11357#discussion_r63577698) [](ancestors = 63577698,63575659)